rest-service/manager_rest/rest/resources_v3/manager.py

```diff
@@ -176,8 +176,8 @@ def _validate_set_ldap_request(self):
             'ldap_dn_extra': {}
         })
         # Not allowing empty username or password
-        ldap_config['ldap_username'] = ldap_config.get('ldap_username') or None
-        ldap_config['ldap_password'] = ldap_config.get('ldap_password') or None
+        ldap_config['ldap_username'] = ldap_config.get('ldap_username') or ''
+        ldap_config['ldap_password'] = ldap_config.get('ldap_password') or ''
```

Comment:
Can be changed to `ldap_config['ldap_password'] = ldap_config.get('ldap_password', '')`